pmm-tests/pg_stat_monitor_setup.sh

```diff
@@ -63,33 +55,51 @@ echo "host    all             all              0.0.0.0/0                       m
 
 sleep 10
 service postgresql restart
-wget https://raw.githubusercontent.com/percona/pmm-qa/main/pmm-tests/tls-ssl-setup/postgres/init.sql
-su postgres bash -c 'psql -f init.sql'
+
+## NOTE
+##I commented following line,THIS SQL SCRIPT IS CAUSING PROBLEMS 
+## As I mentioned earlier postgresql.auto.conf always takes precendence over postgresql.conf 
+## when applying settings. Please use one, either postgresql.conf or postgresql.auto.conf for changes. 
+#wget https://raw.githubusercontent.com/percona/pmm-qa/main/pmm-tests/tls-ssl-setup/postgres/init.sql
+#su postgres bash -c 'psql -f init.sql'
+
 export PATH="/usr/lib/postgresql/${pgsql_version}/bin:$PATH"
 echo $PATH
 cp /usr/lib/postgresql/${pgsql_version}/bin/pg_config /usr/bin
 
 ## Down PGSM repo and move to /home/postgres/pg_stat_monitor dir
 ##
+if [ -z "$pgstat_monitor_branch" ]
+then
+      export pgstat_monitor_branch=REL1_0_STABLE
+fi
+
 cd /home/postgres
 git clone -b ${pgstat_monitor_branch} https://github.com/percona/pg_stat_monitor
 chown -R postgres:postgres pg_stat_monitor
 cd pg_stat_monitor
 
-# Build PGSM 
+# Build PGSM
 make USE_PGXS=1
 
 ## Install build library into server
-## NOTE: YOU NEED SUDO PERMISSION or root user to install into server
 make USE_PGXS=1 install
 
-#$ Stop and restart server to reload PGSM library
-su postgres bash -c 'psql -c "CREATE DATABASE contrib_regression;"'
-su postgres bash -c 'psql -c "ALTER SYSTEM SET shared_preload_libraries TO pg_stat_monitor,pg_stat_statements;" contrib_regression'
 service postgresql stop
-echo "shared_preload_libraries = 'pg_stat_monitor'" | tee -a /etc/postgresql/${pgsql_version}/main/postgresql.conf
+echo "shared_preload_libraries = 'pg_stat_monitor'" >> /etc/postgresql/${pgsql_version}/main/postgresql.conf
+echo "track_activity_query_size=2048"  >> /etc/postgresql/${pgsql_version}/main/postgresql.conf
+echo "track_io_timing=ON"  >> /etc/postgresql/${pgsql_version}/main/postgresql.conf
 service postgresql start
-su postgres bash -c 'psql -d contrib_regression -c "CREATE EXTENSION pg_stat_monitor;"'
-su postgres bash -c 'psql -d postgres -c "CREATE EXTENSION pg_stat_monitor;"'
-su postgres bash -c 'psql -d contrib_regression -c "CREATE EXTENSION pg_stat_statements;"'
-sleep 30
+
+echo "CREATE DATABASE sbtest1;" >> /home/postgres/init.sql
+echo "CREATE DATABASE sbtest2;" >> /home/postgres/init.sql
+echo "CREATE USER pmm WITH PASSWORD 'pmm';" >> /home/postgres/init.sql
+echo "GRANT pg_monitor TO pmm;" >> /home/postgres/init.sql
+su postgres bash -c 'psql -f /home/postgres/init.sql'
+
+service postgresql start
+su postgres bash -c 'psql -c "CREATE DATABASE contrib_regression;"'
```

Comment:
what about creating another database for pg_stat_statement and running similar load on that database

Reply:
Right I forgot that we discuseed it. I will fix it.